### PR TITLE
Update v4.0.0

### PR DESCRIPTION
**Pull request by renan-r-santos**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by renan-r-santos** [comment]
@jan-janssen I have changed `source` from PyPI to GitHub because `4.0.0` has been tagged for a while but not made available on PyPI, see https://github.com/jupyterlab/jupyterlab-latex/issues/218.
Could you please take a look? Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,30 @@
 {% set name = "jupyterlab-latex" %}
-{% set version = "3.1.0" %}
+{% set version = "4.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/j/jupyterlab_latex/jupyterlab_latex-{{ version }}.tar.gz
```
Comment (jan-janssen) [inline recipe/meta.yaml:9]:
```suggestion
  url: https://github.com/jupyterlab/jupyterlab-latex/archive/v{{ version }}.tar.gz
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
-  sha256: f44df285405b5b552b70d10f1f0573d61ca3a4d2ca50bd3097a63a27cdcde16e
```
Comment (jan-janssen) [inline recipe/meta.yaml:10]: Please update the hash to validate the content of the package does not change.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
+  git_url: https://github.com/jupyterlab/jupyterlab-latex.git
+  git_rev: v{{ version }}
```
Comment (jan-janssen) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml (hunk 2 above)
Comment (renan-r-santos) [reply recipe/meta.yaml:10]: Done, tks!

**Review by jan-janssen — APPROVED** [review]
looks good to me

**Pull request merged**